docs/en/basic_concepts/evaluation.md

```diff
@@ -1,193 +1,3 @@
-
-```{math}
-score = 1 - \frac{1}{N}\sum_{i=1}^{N}\frac{D(s_i, \hat{s_{i}})}{max(l_{i},\hat{l_{i}})}
-```
-
-where *N* is the total number of samples, and {math}`D(s_1, s_2)` is the edit distance between two strings.
-
-For example, assume that the real label is "OpenMMLabMMOCR", the prediction of model A is "0penMMLabMMOCR", and the prediction of model B is "uvwxyz". The results of the full matching and `OneMinusNEDMetric` evaluation metrics are as follows:
-
-|         |            |            |
-| ------- | ---------- | ---------- |
-|         | Full-match | 1 - N.E.D. |
-| Model A | 0          | 0.92857    |
-| Model B | 0          | 0          |
-
-As shown in the table above, although the model A only predicted one letter incorrectly, both models got 0 in when using full-match strategy. However, the `OneMinusNEDMetric` evaluation metric can better distinguish the performance of the two models on **long texts**.
-
-### F1Metric
-
-`F1Metric` implements the F1-Metric evaluation metric for KIE tasks and provides two modes, namely `micro` and `macro`.
-
-```python
-val_evaluator = [
-    dict(type='F1Metric', mode=['micro', 'macro'],
-]
-```
-
-- `micro` mode: Calculate the global F1-Metric score based on the total number of True Positive, False Negative, and False Positive.
-
-- `macro` mode：Calculate the F1-Metric score for each class and then take the average.
-
-### Customized Metric
-
-MMOCR supports the implementation of customized evaluation metrics for users who pursue higher customization. In general, users only need to create a customized evaluation metric class `CustomizedMetric` and inherit {external+mmengine:doc}`MMEngine: BaseMetric <design/evaluation>`. Then, the data format processing method `process` and the metric calculation method `compute_metrics` need to be overwritten respectively. Finally, add it to the `METRICS` registry to implement any customized evaluation metric.
-
-```python
-from mmengine.evaluator import BaseMetric
-from mmocr.registry import METRICS
-
-@METRICS.register_module()
-class CustomizedMetric(BaseMetric):
-
-    def process(self, data_batch: Sequence[Dict], predictions: Sequence[Dict]):
-    """ process receives two parameters, data_batch stores the gt label information, and predictions stores the predicted results.
-    """
-        pass
-
-    def compute_metrics(self, results: List):
-    """ compute_metric receives the results of the process method as input and returns the evaluation results.
-    """
-        pass
-```
-
-```{note}
-More details can be found in {external+mmengine:doc}`MMEngine Documentation: BaseMetric <design/evaluation>`.
-```
+Coming Soon!
```

Comment:
Put the tutorials back to Evaluation in order to be consistent with OpenMMLab 2.0 conventions.